activity_hub.php

```diff
@@ -244,19 +261,36 @@ function progress_snapshot_table($show_filtered_projects, $show_filtering_links,
             continue;
         }
 
-        $desired_states = [];
+        echo "\n<tr>";
+        echo "<th class='activity-icon-header'>&nbsp;</th>";
+        echo "<th colspan='3'>" . _("Projects") . " = ";
+        if ($show_filtered_projects) {
+            if ($show_filtering_links) {
+                echo "<a href='?show_filtered=0#progress_snapshot'>" . pgettext("all projects", "All") . "</a> | ";
+            }
+            echo "<b>" . _("Filtered") . "</b>";
+        } else {
+            echo "<b>" . pgettext("all projects", "All") . "</b>";
+            if ($show_filtering_links) {
+                echo " | <a href='?show_filtered=1#progress_snapshot'>" . _("Filtered") . "</a>";
+            }
+        }
+        echo "</tr>\n";
+
+        $desired_states = ["proj_post_first_checked_out"];
```

Comment:
same here on reference to defined states

Reply:
Fixed